iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -300,6 +302,8 @@
 		6EA231702C21BF8900789173 /* RoundedBorderModifier.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = RoundedBorderModifier.swift; sourceTree = "<group>"; };
 		6EA231722C21C58A00789173 /* NearbyTransitPageViewTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = NearbyTransitPageViewTests.swift; sourceTree = "<group>"; };
 		6EA231742C21CF2100789173 /* LoadingCardTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = LoadingCardTests.swift; sourceTree = "<group>"; };
+		6EC6F18C2CE661680022799F /* AlertExtension.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AlertExtension.swift; sourceTree = "<group>"; };
```

Comment:
This is just an aside, but is it possible to get XCode to just keep files in alphabetical order and not commit the file position in the sidebar to git? These diffs cause merge conflicts for me when two people add different files, which defeats a lot of the value of git IMO. If there's a way to move through those quickly that would be great too.

Reply:
Yeah the conflicts can be a pain. We do have a [tech initiative ticket ](https://app.asana.com/0/1205425564113216/1207898757421615/f)for trying out some approaches to address the issue